packages/astro/src/content/utils.ts

```diff
@@ -81,10 +105,10 @@ export async function getEntryData(
 	},
 	collectionConfig: CollectionConfig,
 	shouldEmitFile: boolean,
-	pluginContext: PluginContext
+	pluginContext?: PluginContext
```

Comment:
Why was this changed?

Reply:
So that it can be used outside of a Vite plugin